src/containers/sign-on/store/sagas.js

```diff
@@ -409,6 +422,25 @@ function* signIn(action) {
   }
 }
 
+function* followHotAndNew() {
+  if (process.env.REACT_APP_ENVIRONMENT !== 'production') return
+  yield call(waitForBackendSetup)
+  try {
+    // Hot & New playlist id
+    const hotNewId = 4281
+    const result = yield retrieveCollections(null, [hotNewId])
+
+    if (
```

Comment:
why this check instead of just a truthy/existence check on result?

Reply:
No real reason, just for safety to make sure that the playlist for that id is the one we're expecting